index.js

```diff
@@ -0,0 +1,13 @@
+const tableContainer = document.getElementById('table-container');
```

Comment:
Format code
- new block starts on a new line
- new line inside block add tab space at the start

Pay attention to it, how I format your code
```
const tableContainer = document.getElementById('table-container');
const table = document.createElement('table');
let counter = 1;

for (let i = 0; i < 10; i++) {
  const row = document.createElement('tr');

  for(let j = 0; j < 10; j++) {
    const cell = document.createElement('td');

    cell.textContent = counter++;
    row.appendChild(cell);
  }

  table.append(row);
}

tableContainer.append(table);
```